Library/OcDebugLogLib/DebugPrint.c

```diff
@@ -20,14 +20,17 @@ VOID
 DebugPrintDevicePath (
   IN UINTN                     ErrorLevel,
   IN CONST CHAR8               *Message,
-  IN EFI_DEVICE_PATH_PROTOCOL  *DevicePath
+  IN EFI_DEVICE_PATH_PROTOCOL  *DevicePath  OPTIONAL
```

Comment:
@vit9696 I am proposing making this argument OPTIONAL in this one and the new one, that seemed like the cleanest change. Is that okay?

Reply:
Yes.